### DISCUSSION TITLE
Merge functionality of forms "Add a new IP address" and "Bulk Add IP Addresses"

### DISCUSSION THREAD

**Discussion by SandMouse** [netbox-community/netbox#7682] [Ideas]
When adding IP addresses, using the Web UI, you need to choose between "Add a new IP address" and "Bulk Add IP Addresses". Is it possible to merge the functionality of the two forms? 

The syntax used in the "Address pattern" field of the "Bulk Add IP Addresses" form differs enough from the syntax of a single IP address in the "Add a new IP address" form. This way you can distinguish creating a set of IP addresses and creating a single IP address entry?

**Comment by jeremystretch** [comment]
> Is it possible to merge the functionality of the two forms?

Not really. When adding a single IP address, the user has the option of assigning it to a particular interface. This is not the case when adding multiple IPs simultaneously. The address fields are also validated and processed differently, as the bulk form supports range assignment.
